tool-base/build.gradle.kts

```diff
@@ -24,19 +24,20 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import io.spine.gradle.internal.IncrementGuard
 import io.spine.internal.dependency.JavaPoet
 import io.spine.internal.dependency.JavaX
 
 group = "io.spine.tools"
```

Comment:
It looks like this is a default group for all of the modules here. Can we move this declaration to the root-level `subprojects` config then>?